compiler/circle-tensordump/driver/Driver.cpp

```diff
@@ -49,6 +49,13 @@ int entry(int argc, char **argv)
     return 255;
   }
 
+  if (arser["--tensors_to_hdf5"] == arser["--tensors"])
```

Comment:
I wonder what `arser["--tensors_to_hdf5"]` and `arser["--tensors"]` return... :)

Reply:
@seanshpark That is a boolean value for whether the option has been passed or not.